### PR TITLE
Duplicate checking, open now

### PR DESCRIPTION
**Pull request by sandlerben**

### CODE REVIEW

File: app/utils.py [hunk 1]
```diff
@@ -155,14 +155,11 @@ def is_open(resource, current_date = None):
...
-	pair_bool = False
 	for pair in month_pairs:
 		start_month = pair.start_month
 		end_month = pair.end_month
-		pair_bool = pair_bool or (current_date.month <= end_month and current_date.month >= start_month)
-
-	if(not pair_bool):
-		return False
+		if not (current_date.month <= end_month and current_date.month >= start_month):
+			return False
```
Comment (weronica) [inline app/utils.py]: We aren't using `OpenMonthPair`s anymore. Could you remove this check?

File: app/utils.py [hunk 2]
```diff
@@ -433,6 +433,20 @@ def check_time_format(field_val, row_index):
...
+					description = description,
+					are_hours_available = are_hours_available,
+					is_for_family_and_children = is_for_family_and_children,
+					is_for_seniors = is_for_seniors,
+					is_wheelchair_accessible = is_wheelchair_accessible,
+					is_accepts_snap = is_accepts_snap).first()
+
+				if duplicate:
+					make_error("Identical resource already exists in database.", i)
+
```
Comment (weronica) [inline app/utils.py]: You might note in the error which resource is a duplicate. For example, "Identical resource (Clark Park) already exists in the database."

File: app/utils.py (hunk 1 above)
Comment (sandlerben) [inline app/utils.py]: Sure.

File: app/utils.py (hunk 2 above)
Comment (sandlerben) [inline app/utils.py]: Okay.

**Pull request merged**